include/cantera/zeroD/ConstPressureMoleReactor.h

```diff
@@ -0,0 +1,46 @@
+//! @file ConstPressureMoleReactor.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_CONSTPRESSMOLE_REACTOR_H
+#define CT_CONSTPRESSMOLE_REACTOR_H
+
+#include "cantera/zeroD/MoleReactor.h"
+
+namespace Cantera
+{
+
+/*!
+ * ConstPressureMoleReactor is a class for constant-pressure reactors
+ * which use a state of moles.
+ * @SInCE New in Cantera 3.0
```

Comment:
GitHub is really making a mess out of this. The annotation command should just be lowercase `since` preceded by an `@`, like every other Doxygen command. The funky capitalization came from GitHub deciding that this was _clearly_ a mention of a user who has their name capitalized that way.

Reply:
Ah, I assumed it was just rendered that way and not committed that way. Making these changes now.